test/e2e/network/netpol/policies.go

```diff
@@ -244,6 +244,26 @@ func GetAllowIngressByNamespaceOrPod(name string, targetLabels map[string]string
 	return policy
 }
 
+// GetAllowIngressByMultiLabeledPod allows ingress for pods with matching multiple pod labels
+func GetAllowIngressByMultiLabeledPod(name string, targetLabels map[string]string, peers []networkingv1.NetworkPolicyPeer) *networkingv1.NetworkPolicy {
```

Comment:
This name makes me think that the pod needs to have multiple labels in order to be matched -- whereas the intent is that the pod needs to have any of the labels in order to match, right?  Can the name be updated to clarify that?

Reply:
That's correct, the labels are treated as OR, maybe `GetAllowIngressByAnyPod`, this follows the semantics of using Pod for PodSelectors, like `// GetAllowIngressByPod allows ingress by pod labels`